docs/related/amf-conversion.mdx

```diff
@@ -1,9 +1,193 @@
 ---
 id: amf_conversion
 title: AMF Conversion
-hide_title: true
 ---
 
-# AMF Conversion
+:::caution
+The behaviour expressed in this documentation is currently a work in progress.
+If any bugs are found that do not satisfy these requirements,
+please file an issue in the [aml-org/amf repository](https://github.com/aml-org/amf/pull/715).
+:::
 
+# What is WebAPIs conversion?
 
+Conversion stands for the capability of parsing an API from one specification vendor into an API graph and then rendering said graph as a different vendor’s specification.
+
+Different specifications can have different required fields, and the same information conceptually may not be described in the same way.
+This is the reason why in a *cross parsing-rendering process*, the AMF graph needs to go through a transformation called *“Compatibility resolution”*.
+
+**Compatibility resolution** is a specific resolution pipeline that is used to adapt an API graph parsed from spec A, to fulfill the requirements of the spec in which it will be rendered.
```

Comment:
It may be unclear what "spec A" is referring to, since there's no mention of a "spec B" here.

Maybe:
**Compatibility resolution** is a specific resolution pipeline that is used to adapt an API graph parsed from one spec, to fulfill the requirements of another spec in which it will be rendered.

Reply:
Thank you, I've written it again with your suggestion.